tests/benchmarks/test_futures.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+from dask.distributed import as_completed, wait
+from distributed.utils_test import inc, slowdec, slowinc
+
+
+def test_single_future(small_client):
+    """How quickly can we run a simple computation?"""
+    small_client.submit(inc, 1).result()
+
+
+def test_large_map(small_client):
+    """What's the overhead of map these days?"""
+    futures = small_client.map(inc, range(100_000))
+    wait(futures)
+
+
+def test_large_map_first_work(small_client):
+    """
+    Large maps are fine, but it's pleasant to see work start immediately.
+    We have a batch_size keyword that should work here but it's not on by default.
+    Maybe it should be.
+    """
+    futures = small_client.map(inc, range(100_000))
+    for _ in as_completed(futures):
+        return
```

Comment:
return?

Reply:
Yup.  I just want to know when the first thing has completed.